lib/attack.go

```diff
@@ -192,15 +202,23 @@ func HTTP2(enabled bool) func(*Attacker) {
 	}
 }
 
+func HTTP3(enabled bool) func(*Attacker) {
+	return func(a *Attacker) {
+		if enabled {
+			a.client.Transport = &http3.RoundTripper{}
```

Comment:
Any chance we can "translate" already set configuration in an existing Transport to this `http3.RoundTripper`? Akin to what `http2.ConfigureTransport(tr)` does. Example: Not loosing existing `TLSClientConfig`.